src/main/java/seedu/internbuddy/storage/JsonAdaptedCompany.java

```diff
@@ -15,54 +15,58 @@
 import seedu.internbuddy.model.company.Email;
 import seedu.internbuddy.model.company.Name;
 import seedu.internbuddy.model.company.Phone;
+import seedu.internbuddy.model.company.Status;
 import seedu.internbuddy.model.tag.Tag;
 
 /**
  * Jackson-friendly version of {@link Company}.
  */
-class JsonAdaptedCompany {
-
-    public static final String MISSING_FIELD_MESSAGE_FORMAT = "company's %s field is missing!";
+public class JsonAdaptedCompany {
+    public static final String MISSING_FIELD_MESSAGE_FORMAT = "Company's %s field is missing!";
 
     private final String name;
     private final String phone;
     private final String email;
     private final String address;
+    private final String status;
     private final List<JsonAdaptedTag> tags = new ArrayList<>();
 
     /**
-     * Constructs a {@code JsonAdaptedcompany} with the given company details.
+     * Constructs a {@link JsonAdaptedCompany} with the given person details.
```

Comment:
Why the change in the Javadoc for this constructor as well as the reference to person?